src/components/calculator/ActivityRiskControls.tsx

```diff
@@ -51,6 +51,15 @@ export const ActivityRiskControls: React.FunctionComponent<{
           <Trans>calculator.precautions.no_intimate_bonus_outdoors</Trans>
         </div>
       ) : null}
+      {data.setting === 'filtered' && (
+        <div className="warning">
+          <Trans>calculator.precautions.filtered_ensure_airflow</Trans>{' '}
+          <Link to="/paper/14-research-sources#ventilation">
+            <Trans>calculator.precautions.filtered_ensure_airflow_link</Trans>
```

Comment:
I've been learning way more about translation than I ever knew. One pro tip: don't assume grammer structure. Some languages might not end on the word "here". Instead, you can put the link inside a single <Trans> element. Then in the en.json file you can follow the pattern you see elsewhere of using <2>Foo</2> to fill in the link tag.

Let me know if you have any questions.

Reply:
Done! That's a really cool feature.